README.md

```diff
@@ -17,10 +17,10 @@ their respective handler.
 
 The name mux stands for "HTTP request multiplexer". Like the standard `http.ServeMux`, `mux.Router` matches incoming requests against a list of registered routes and calls a handler for the route that matches the URL or other conditions. The main features are:
 
-* It implements the `http.Handler` interface so it is compatible with the standard `http.ServeMux`.
+* It implements the `http.Handler` interface so, it is compatible with the standard `http.ServeMux`.
 * Requests can be matched based on URL host, path, path prefix, schemes, header and query values, HTTP methods or using custom matchers.
 * URL hosts, paths and query values can have variables with an optional regular expression.
-* Registered URLs can be built, or "reversed", which helps maintaining references to resources.
+* Registered URLs can be built, or "reversed", which helps to maintaining references to resources.
```

Comment:
I believe it should be either `to maintain` or `in maintaining`. Please check.

Reply:
done